### PR TITLE
updated babel dependencies

### LINKED ISSUE #286: Update babel dependencies
It looks like the babel dependencies have been upgraded.

```
babel-cli    6.16.0  →        6.18.0
babel-core    6.17.0  →        6.18.0
babel-loader    6.2.5  →         6.2.7
babel-plugin-transform-react-remove-prop-types     0.2.9  →        0.2.10
```

Please check out the changelogs for these tools and make sure that there are not breaking changes that may affect kyt. Also, it would be nice to test this against the Universal starter kyt, along with the e2e and unit tests.

### PR DESCRIPTION
**Pull request by PedroCSSilva**
I had some trouble trying to setup the project and help but I think everything is ok.

kyt probably needs better contributing guidelines for people like me who are just getting started.

closes #286

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "kyt",
   "version": "0.3.0",
   "description": "kyt is a toolkit that encapsulates and manages the configuration for web apps.",
-  "main": "",
+  "main": "index.js",
```
Comment (delambo) [inline package.json]: Can you change this to `cli/index.js` please?

**Comment by delambo** [comment]
Also, can you merge in the latest.

**Comment by PedroCSSilva** [comment]
What do you mean? Sorry I'm still learning.

**Comment by delambo** [comment]
Try these instructions [here](https://help.github.com/articles/merging-an-upstream-repository-into-your-fork/). There's a conflict in package.json, so follow the link in 5) to resolve the conflict. Let me know if you get stuck.

**Comment by PedroCSSilva** [comment]
I looked at that link but git status tell me I'm up to date. Can you show me your package.json? May be because I'm having trouble setting uo the project with kyt.

**Pull request closed without merging**